### PR TITLE
Bench pressing makes you not fat anymore.

### PR DESCRIPTION
**Pull request by Xander3359**
## About The Pull Request
If you bench press you become not fat.
## Why It's Good For The Game
Credit akimomi on discord:
"Fat slowdown isn't a good mechanic, it's not enjoyable for marines and doesn't add anything to the game except as a way to punish new players for eating food and people taking advantage of the ability to have infinite nutrition.
It doesn't fit for TGMC."

Someone then said lifting should remove fat so here we go.
## Changelog
:cl:
add: Bench press is a great way to lose weight
/:cl:

### CODE REVIEW

**Comment by novaepee** [comment]
6ddcbadb2d44d8bf721421eee37c655071af6a9e943c1507f2365965d7cee9e2

**Comment by FIXTHERADBOXPLEASE** [comment]
Can confirm this works in real life too.

**Review by QualityVan — APPROVED** [review]
I'm wondering if this'd fit better with a flat reduction instead of immediate drop to cap, but that's nitpicking and it's fine as is.

File: code/game/objects/structures/benchpress.dm [hunk 1]
```diff
@@ -127,6 +127,10 @@
 		ADD_TRAIT(user, TRAIT_WORKED_OUT, WEIGHTBENCH_TRAIT)
 		addtimer(CALLBACK(src, PROC_REF(undo_buff), WEAKREF(user)), 15 MINUTES)
 	var/finishmessage = pick("You feel stronger!","You feel like you're the boss of this gym!","You feel robust!","The challenge is real!")
+	var/mob/living/carbon/bencher = user
+	if(bencher.nutrition >= NUTRITION_OVERFED)
+		bencher.set_nutrition(NUTRITION_OVERFED-1)
+		finishmessage = pick("You no longer feel overweight.","You clothes are no longer too tight.","YOU BECOME LESS FAT!")
```
Comment (TiviPlus) [inline code/game/objects/structures/benchpress.dm:133]: Why are you setting instead of using nutrition

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Pull request merged**